src/App.js

```diff
@@ -1,16 +1,57 @@
 import React from 'react';
 import './App.css';
+import { useState } from 'react';
```

Comment:
To make your file even shorten you can combine imports like so: `import React, { useState } from 'react';`